code/week3/week3.cpp

```diff
@@ -1,274 +0,0 @@
-#include "week3.h"
```

Comment:
Why is this being deleted?

Reply:
I think this is our screwed branch management at work

Reply:
Yes, I believe this, and other weirdness in this PR, is because week3hardware has the latest stuff from master, but weekThree does not.